regression/esbmc-solidity/github_bool_literal/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+contract.solast
+--function foo --contract contract.sol
+^VERIFICATION SUCCESSFUL$
```

Comment:
Would it be possible to add another regression test that reports "VERIFICATION FAIL"? 
Ideally, we woud like two test cases one reporting successful and the other reporting failed.